Packs/DigitalGuardian/Integrations/DigitalGuardianARCEventCollector/DigitalGuardianARCEventCollector_test.py

```diff
@@ -1,123 +1,152 @@
 
-    raw_events = util_load_json('test_data/events_mock_request.json')
-    mocker.patch.object(Client, 'get_token', return_value='token')
-    mocker.patch.object(Client, 'get_events', return_value=raw_events)
-    client = Client(verify=False, proxy=False, auth_url="example.com", gateway_url="test.com", base_url="exmpt.com",
-                    client_id="11", client_secret="22", export_profile="33")
-    events = get_raw_events(client, None)
+    raw_response = util_load_json('test_data/mock_response.json')  # contains duplicate events
+    mocker.patch.object(authenticated_client, 'export_events', return_value=raw_response)
+    expected_events = util_load_json('test_data/expected_events.json')
 
-    mock_events = util_load_json('test_data/events.json')
+    events, last_run = fetch_events(authenticated_client)
+    outputted_events = list(events)
 
-    assert events == mock_events
+    assert len(raw_response['data']) >= len(outputted_events)  # sanity check (duplicates should be removed)
+    assert outputted_events == expected_events
+    assert last_run['bookmark_values'] == raw_response['bookmark_values']
+    assert last_run['search_after_values'] == raw_response['search_after_values']
 
 
-def test_get_events_command(mocker):
+def test_get_events_command(mocker: MockerFixture, authenticated_client: Client):
     """
     Given:
         - Digital Guardian ARC client and limit of events to get
     When:
-        - Calling get_events command, which will run after the get_raw_events and will return results according to the
-          limit that was provided
+        - Calling get_events_command
     Then:
-        - Ensure the events are returned as expected
+        - Ensure the events are returned as expected and correct arguments are passed to tableToMarkdown
     """
-    from DigitalGuardianARCEventCollector import Client, get_events_command
+    from DigitalGuardianARCEventCollector import get_events_command
 
-    raw_events = util_load_json('test_data/events_mock_request.json')
-    mocker.patch.object(Client, 'get_token', return_value='aaa')
-    mocker.patch.object(Client, 'get_events', return_value=raw_events)
+    limit = 1
+    raw_response = util_load_json('test_data/mock_response.json')
+    mocker.patch.object(authenticated_client, 'export_events', return_value=raw_response)
+    table_to_markdown = mocker.patch('DigitalGuardianARCEventCollector.tableToMarkdown')
 
-    args = {"limit": 1}
-    client = Client(verify=False, proxy=False, auth_url="example.com", gateway_url="test.com", base_url="exmpt.com",
-                    client_id="11", client_secret="22", export_profile="33")
-    events, _ = get_events_command(client, args)
+    outputted_events, *_ = get_events_command(authenticated_client, args={'limit': limit})
 
-    expected_events = util_load_json('test_data/events_mock_1_response.json')
+    expected_events = util_load_json('test_data/expected_events.json')[:limit]  # slice list to length = limit
```

Comment:
```suggestion
    expected_events = util_load_json('test_data/expected_events.json')[:limit]
```

Reply:
Removed comment in commit 582763c169821e56b0a063bc42e00239eb151efc